appmap/test/data/unittest/simple/simple.py

```diff
@@ -0,0 +1,9 @@
+class Simple:
+    def hello(self):
+        return 'Hello'
+
+    def world(self):
+        return 'world'
+
+    def hello_world(self, bang):
+        return '%s %s%s' % (self.hello(), self.world(), bang)
```

Comment:
It's not very important, but I'll note this could go into `__init__` module, leading to better test names, `simple.Simple` vs. `simple.simple.Simple`. Simpler! ;)